app/src/main/java/org/kinecosystem/kinit/view/feedback/FeedbackWebFragment.kt

```diff
@@ -0,0 +1,68 @@
+package org.kinecosystem.kinit.view.feedback
+
+import android.annotation.SuppressLint
+import android.content.pm.PackageManager
+import android.databinding.DataBindingUtil
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.RenderProcessGoneDetail
+import android.webkit.WebSettings
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import org.kinecosystem.kinit.BuildConfig
+import org.kinecosystem.kinit.R
+import org.kinecosystem.kinit.databinding.FeedbackLayoutFragmentBinding
+import org.kinecosystem.kinit.view.BaseFragment
+import org.kinecosystem.kinit.viewmodel.FeedbackViewModel
+
+
+class FeedbackWebFragment : BaseFragment(){
```

Comment:
Do we really need 2 WebFragment classes and 2 models? Could we not create a single one that takes care of both the Feedback and Support? 
There is a lot of common functionality. What are all the differences?

Reply:
Yes I was thinking of merging it into one.